tasks/icinga2-Debian.yml

```diff
@@ -24,6 +24,6 @@
   become: yes
   apt:
     name: icinga2
-    state: present
+    state: latest
```

Comment:
Upgrading icinga2 can have breaking changes so I'd prefer to keep this to `latest` to avoid breaking idempotency.

Reply:
True. I had changed this to upgrade from Debian's 2.6 to Icinga's 2.10. Having had the wrong release ID previously. I will change this back.